ingestfile

```diff
@@ -259,7 +292,7 @@ fi
         read NOTVIDRESPONSE
         [ "${NOTVIDRESPONSE}" == "q" ] && exit 0
     fi
-    if [ "${AUDIODECISION}" = "" ] ; then
+    if [ "${AUDIODECISION}" = "" ] && [ "${SILENT_RUNNING}" == "" ] ; then
```

Comment:
I would suggest to use either `=` or `==` for tests, but not a mix of both, because it’s less easy to read and to maintain.

Reply:
Thanks, I'll change those!